src/keys/public.rs

```diff
@@ -64,3 +64,10 @@ impl Serializable<32> for PublicKey {
         Ok(Self(JubJubAffine::from_bytes(bytes)?.into()))
     }
 }
+
+impl PublicKey {
+    /// Create a public key from its internal parts
+    pub const fn from_raw_unchecked(key: JubJubExtended) -> Self {
```

Comment:
Consider commenting here on what "unchecked" means. Something saying what is not being checked, why it should be used with care (if applicable), etc...

Reply:
Agreed, updated